### PR TITLE
Add text-wrap and white-space-collapse, and update white-space

### PR DESCRIPTION
**Pull request by chrisdavidmills**
### Description

Chrome 114 adds the `text-wrap` and `white-space-collapse` CSS properties, and updates the `white-space` property with new behavior — as well as taking its usual keywords, it can now act as a shorthand property for `text-wrap` and `white-space-collapse`. These items are specified in [CSS Text Level 4](https://www.w3.org/TR/css-text-4/).

This PR adds those items to the data. I also took this opportunity to add `.DS_Store` files to the `.gitignore` file. As a mac user, that was getting on my nerves ;'-)

Note - there is another related property, `white-space-trim`, which is specced as being included in the `white-space` shorthand value, but this is not supported yet so I have not added it. Let me know if you think I should add it.

### Additional details

My [research document](https://docs.google.com/document/d/1de7T6UbSEcDvKKmyqz-yS3D2CS7XT67mxFjHhzKYlMc/edit#) provides additional details about this overall project and its required changes.

### CODE REVIEW

**Comment by chrisdavidmills** [comment]
@bsmth can you review this one please mate?

**Comment by bsmth** [comment]
I can indeed, I'll take a look tomorrow if that's okay

**Comment by chrisdavidmills** [comment]
> I can indeed, I'll take a look tomorrow if that's okay

That's perfect, thanks!

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
 /node_modules/
-.DS_Store
+.DS_Store
```
Comment (bsmth) [inline .gitignore:2]: 🤔 I thought I added this here: https://github.com/mdn/data/commit/4f74b576de422743b13752064355857610c3b4fc not sure what the difference is

Comment (chrisdavidmills) [reply .gitignore:2]: I added it too, in this PR, so it ended up creating a merge conflict. That's why it looks a little odd.

File: css/properties.json [hunk 2]
```diff
@@ -9851,7 +9867,7 @@
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/visibility"
   },
   "white-space": {
-    "syntax": "normal | pre | nowrap | pre-wrap | pre-line | break-spaces",
+    "syntax": "normal | pre | nowrap | pre-wrap | pre-line | break-spaces | [ <'white-space-collapse'> || <'text-wrap'> ]",
```
Comment (bsmth) [inline css/properties.json:9870]: From your notes:

>there is another related property, white-space-trim, which is specced as being included in the white-space shorthand value, but this is not supported yet so I have not added it. Let me know if you think I should add it.

My opinion is to add it here for completeness sake, and we're going to have to add it in future anyway. Protocol on that looks like something in BCD (a note?) for browser-specific details, what do you reckon?

Adding the suggestion for your convenience:

```suggestion
    "syntax": "normal | pre | nowrap | pre-wrap | pre-line | break-spaces | [ <'white-space-collapse'> || <'text-wrap'> || <'white-space-trim'> ]",
```

spec link: https://www.w3.org/TR/css-text-4/#white-space-property

**Review by bsmth — APPROVED** [review]
Looks good, @chrisdavidmills - one comment about the missing prop you mentioned, but ready to go after that 👍🏻

**Comment by chrisdavidmills** [comment]
> Looks good, @chrisdavidmills - one comment about the missing prop you mentioned, but ready to go after that 👍🏻

Cool, thanks for the review @bsmth . I've added `white-space-trim`, as suggested.

**Comment by chrisdavidmills** [comment]
@bsmth ooop, I can't merge in this repo. Can you merge it? Ta.

File: .gitignore (hunk 1 above)
Comment (bsmth) [reply .gitignore:2]: Gotcha

**Pull request merged**